docs/install.md

```diff
@@ -202,8 +202,8 @@ Once the above deployment installation is successful, we can see all the driver
 pods in `Running` state:
 ``` console
 $ kubectl get deployments.pmem-csi.intel.com
-NAME                 AGE
-pmem-deployment      50s
+NAME                 DEVICEMODE   NODESELECTOR   IMAGE   AGE   STATUS
+pmem-deployment      lvm                                 50s   Running
```

Comment:
Other types list the age as last field:
```
$ kubectl get all --all-namespaces
NAMESPACE     NAME                                                    READY   STATUS    RESTARTS   AGE
kube-system   pod/coredns-66bff467f8-tnshp                            1/1     Running   0          122m
kube-system   pod/coredns-66bff467f8-tpstn                            1/1     Running   0          122m
kube-system   pod/etcd-pmem-csi-pmem-govm-master                      1/1     Running   0          122m
kube-system   pod/kube-apiserver-pmem-csi-pmem-govm-master            1/1     Running   0          122m
kube-system   pod/kube-controller-manager-pmem-csi-pmem-govm-master   1/1     Running   0          122m
kube-system   pod/kube-proxy-5gbns                                    1/1     Running   0          122m
kube-system   pod/kube-proxy-bq86z                                    1/1     Running   0          122m
kube-system   pod/kube-proxy-gf4hj                                    1/1     Running   0          122m
kube-system   pod/kube-proxy-ggtxp                                    1/1     Running   0          122m
kube-system   pod/kube-scheduler-pmem-csi-pmem-govm-master            1/1     Running   0          122m
kube-system   pod/weave-net-5h9sb                                     2/2     Running   1          122m
kube-system   pod/weave-net-l4pkw                                     2/2     Running   1          122m
kube-system   pod/weave-net-rb8tk                                     2/2     Running   0          122m
kube-system   pod/weave-net-rj4zq                                     2/2     Running   0          122m

NAMESPACE     NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                  AGE
default       service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP                  122m
kube-system   service/kube-dns     ClusterIP   10.96.0.10   <none>        53/UDP,53/TCP,9153/TCP   122m

NAMESPACE     NAME                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
kube-system   daemonset.apps/kube-proxy   4         4         4       4            4           kubernetes.io/os=linux   122m
kube-system   daemonset.apps/weave-net    4         4         4       4            4           <none>                   122m

NAMESPACE     NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
kube-system   deployment.apps/coredns   2/2     2            2           122m

NAMESPACE     NAME                                 DESIRED   CURRENT   READY   AGE
kube-system   replicaset.apps/coredns-66bff467f8   2         2         2       122m
```

IMHO we should follow that.

Reply:
Ok. Pushed the change.